extensions/wikia/Wall/WallMessage.class.php

```diff
@@ -162,7 +162,7 @@ public function setOrderId( $val = 1 ) {
 	/**
 	 * @return CommentsIndexEntry
 	 */
-		public function getCommentsIndexEntry() {
+	public function getCommentsIndexEntry() {
 		if ( false === $this->commentsIndex ) { // false means we didn't call newFromId yet
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)